src/api/l_physics_shapes.c

```diff
@@ -246,6 +246,36 @@ static int l_lovrShapeSetOrientation(lua_State* L) {
   return 0;
 }
 
+static int l_lovrShapeGetPose(lua_State* L) {
+  Shape* shape = luax_checkshape(L, 1);
+  float x, y, z;
+  lovrShapeGetPosition(shape, &x, &y, &z);
+  float angle, ax, ay, az, orientation[4];
+  lovrShapeGetOrientation(shape, orientation);
+  quat_getAngleAxis(orientation, &angle, &ax, &ay, &az);
+  lua_pushnumber(L, x);
+  lua_pushnumber(L, y);
+  lua_pushnumber(L, z);
+  lua_pushnumber(L, angle);
+  lua_pushnumber(L, ax);
+  lua_pushnumber(L, ay);
+  lua_pushnumber(L, az);
+  return 7;
+}
+
+static int l_lovrShapeSetPose(lua_State* L) {
+  Shape* shape = luax_checkshape(L, 1);
+  lovrAssert(lovrShapeGetCollider(shape) != NULL, "Shape must be attached to collider");
+
+  float position[3];
+  luax_readvec3(L, 2, position, NULL);
+  lovrShapeSetPosition(shape, position[0], position[1], position[2]);
+  float orientation[4];
+  luax_readquat(L, 5, orientation, NULL);
```

Comment:
```suggestion
  int index = luax_readvec3(L, 2, position, NULL);
  lovrShapeSetPosition(shape, position[0], position[1], position[2]);
  float orientation[4];
  luax_readquat(L, index, orientation, NULL);
```

This change will make it work with both vec3 and numbers.

Reply:
OK, Fixed